generator/operator.go

```diff
@@ -1,14 +1,200 @@
 
+import (
+	"net/http"
+
+	"github.com/hashicorp/nomad/api"
+)
+
 func (v *v1api) getOperatorPaths() []*apiPath {
-	// tags := []string{"Operator"}
+	tags := []string{"Operator"}
 
 	return []*apiPath{
 		//s.mux.HandleFunc("/v1/operator/license", s.wrap(s.LicenseRequest))
+		// NOTE: there's no license endpoint
+		// QUESTION: should this be under the enterprise-api instead...?
 		//s.mux.HandleFunc("/v1/operator/raft/", s.wrap(s.OperatorRequest))
+		{
+			Template: "/operator/raft/configuration",
+			Operations: []*operation{
+				newOperation(http.MethodGet,
+					httpServer.OperatorRequest,
+					tags,
+					"GetOperatorRaftConfiguration",
+					nil,
+					defaultQueryOpts,
+					newResponseConfig(200,
+						objectSchema,
+						// NOTE: it should be RaftConfigurationResponse
+						// however that doesn't exist and is likely due
+						// to a naming error
+						api.RaftConfiguration{},
+						nil,
+						"GetOperatorRaftConfigurationResponse",
+					),
+				),
+			},
+		},
+		{
+			Template: "/operator/raft/peer",
+			Operations: []*operation{
+				newOperation(http.MethodDelete,
+					httpServer.OperatorRequest,
+					tags,
+					"DeleteOperatorRaftPeer",
+					nil,
+					defaultWriteOpts,
+					newResponseConfig(200,
+						nilSchema,
+						nil,
+						nil,
+						"DeleteOperatorRaftPeerResponse",
+					),
+				),
+			},
+		},
 		//s.mux.HandleFunc("/v1/operator/autopilot/configuration", s.wrap(s.OperatorAutopilotConfiguration))
+		{
+			Template: "/operator/autopilot/configuration",
+			Operations: []*operation{
+				newOperation(http.MethodGet,
+					httpServer.OperatorAutopilotConfiguration,
+					tags,
+					"GetOperatorAutopilotConfiguration",
+					nil,
+					defaultQueryOpts,
+					newResponseConfig(200,
+						objectSchema,
+						// NOTE: return struct defined at the bottom and
+						// does not use api.AutopilotConfiguration{}
+						AutopilotConfiguration{},
+						nil,
+						"GetOperatorAutopilotConfigurationResponse",
+					),
+				),
+				// Does not accept POST -- only PUT
+				newOperation(http.MethodPut,
+					httpServer.OperatorAutopilotConfiguration,
+					tags,
+					"PutOperatorAutopilotConfiguration",
+					newRequestBody(objectSchema, api.AutopilotConfiguration{}),
+					defaultWriteOpts,
+					newResponseConfig(200,
+						boolSchema,
+						true,
+						nil,
+						"PutOperatorAutopilotConfigurationResponse",
+					),
+				),
+			},
+		},
 		//s.mux.HandleFunc("/v1/operator/autopilot/health", s.wrap(s.OperatorServerHealth))
+		{
+			Template: "/operator/autopilot/health",
+			Operations: []*operation{
+				newOperation(http.MethodGet,
+					httpServer.OperatorServerHealth,
+					tags,
+					"GetOperatorAutopilotHealth",
+					nil,
+					defaultQueryOpts,
+					newResponseConfig(200,
+						objectSchema,
+						api.OperatorHealthReply{},
+						nil,
+						"GetOperatorAutopilotHealthResponse",
+					),
+				),
+			},
+		},
 		//s.mux.HandleFunc("/v1/operator/snapshot", s.wrap(s.SnapshotRequest))
+		/* {
```

Comment:
You are still working on this right?  I notice it's in draft mode and just want to confirm.

Reply:
I'm quite stuck with the snapshots.  Not sure how to handle the streaming outputs.  Will need your help on this one, @DerekStrickland

Reply:
You can safely skip streaming endpoints for now.  Thanks for trying to add them, but putting a bit TODO comment here would be fine for now.

For background, there is a long running [open issue](https://github.com/OAI/OpenAPI-Specification/issues/1576) around streaming data in the OpenAPI spec. OpenAPI is really built for REST/RPC which don't exactly map to the streaming model. There is an [alternative standard called AsyncAPI](https://www.asyncapi.com/) based off the OpenAPI spec. We [plan to experiment](https://github.com/hashicorp/nomad-openapi/blob/7cb6f5d550d9b18609c2aea25da07723eaea9882/v1/README.md#goals) with using AsyncAPI, but it's not part of our first milestone.

Reply:
Oh that's right!  I totally forgot!  That was one of the things that was mentioned in the early monthly meetings.  Okay, I will add a TODO note to `generator/operator.go` and this branch should be ready to be merged!